src/bunit.core/Extensions/RenderedComponentRenderExtensions.cs

```diff
@@ -30,7 +30,23 @@ public static void SetParametersAndRender<TComponent>(this IRenderedComponentBas
 			throw new ArgumentNullException(nameof(renderedComponent));
 
 		var renderer = renderedComponent.Services.GetRequiredService<TestRenderer>();
-		renderer.SetDirectParameters(renderedComponent, parameters);
+		var result = renderer.SetDirectParametersAsync(renderedComponent, parameters);
```

Comment:
```suggestion
try
{
  renderer.SetDirectParametersAsync(renderedComponent, parameters).GetAwaiter().GetResult();
}
catch (AggregateException e) when (e.InnerException.Count == 1)
{
  ExceptionDispatchInfo.Capture(e.InnerExceptions[0]).Throw();
}
```

Fits more to what we already have.